brightray/browser/browser_main_parts.cc

```diff
@@ -87,6 +95,27 @@ void OverrideLinuxAppDataPath() {
   PathService::Override(DIR_APP_DATA, path);
 }
 
+void OverrideWinAppLogsPath() {
+  std::string appName = GetApplicationName();
+  std::string logPath = "%HOMEDRIVE%%HOMEPATH%\\AppData\\Roaming\\";
+  std::string appLogPath = logPath + appName + "\\logs";
+
+  int status = mkdir(appLogPath.c_str(), S_IRWXU | S_IRGRP | S_IROTH);
+
+  PathService::Override(DIR_APP_LOGS, base::FilePath(appLogPath));
+}
+
+void OverrideLinuxAppLogsPath() {
+  std::string appName = GetApplicationName();
+  std::string homePath = std:string(getenv("HOME"));
+  std::string appLogPath = homePath + "/.config/" + appName + "/logs";
```

Comment:
We're gonna get users asking this to respect `$XDG_CONFIG_HOME `, current envs don't though (and there is an issue for it).  So do what you will 😆